src/Menus.cpp

```diff
@@ -1374,7 +1362,6 @@ void showSettingsMenu(char* menuTitle) {
     stopMenu = checkStopMenu(startTime);
 
     current_selection = display.userInterfaceSelectionList(menuTitle, current_selection, _menu);
-    uint8_t fnc = menuOrdinals[current_selection];
```

Comment:
Although you're saving one byte of stack memory, I might need the function code for further operations in upcoming versions.